src/Core/Grid/Definition/Factory/AttributeGroupGridDefinitionFactory.php

```diff
@@ -200,7 +200,7 @@ protected function getFilters()
             )
             ->add((new Filter('position', ReorderPositionsButtonType::class))
                 ->setAssociatedColumn('position')
-            )
+            )          
```

Comment:
@mattgoud you just need to remove these extra spaces and it should be good
Maybe you can squash your commits also